File: Hishab Server/hishabApp/hishabApp/src/main/java/com/example/hishab/hishabApp/model/UserModel.java
package com.example.hishab.hishabApp.model;


import javax.persistence.*;

import lombok.Data;

@Data
@Entity
@Table(name = "tbl_user")
public class UserModel {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int userId;
    private String mobileNo;
    private String email;
    private String name;
    private String photoUrl;
    private String password;
}

File: Hishab/app/src/main/java/com/example/hishab/retrofit/commonmodel/UserModel.kt
package com.example.hishab.retrofit.commonmodel;

import com.example.hishab.retrofit.JsonConverter
import com.example.hishab.utils.Constant
import com.example.hishab.utils.PreferenceHelper

data class UserModel(var userId: Int, var mobileNo: String, var name: String, var email: String) :JsonConverter() {
    var password: String? = null
    var photoUrl: String? = null

    fun saveInfoToPreference(){
        PreferenceHelper.save(Constant.User_Id,userId);
        PreferenceHelper.save(Constant.MobileNO,mobileNo);
        PreferenceHelper.save(Constant.UserName,name);
        PreferenceHelper.save(Constant.email,email)
        PreferenceHelper.save(Constant.photoUrl,photoUrl)
    }
}
